src/navigator.js

```diff
@@ -17,6 +36,11 @@ class Navigator {
 
           const { latitude, longitude, accuracy } = position.coords;
 
+          if (locationTimeToLive) {
+            const lastLocation = { latitude, longitude, accuracy, updatedAt: Date.now() };
+            Storage.setItem(Storage.LAST_LOCATION, JSON.stringify(lastLocation));
+          }
```

Comment:
It probably makes sense to bake in the expiration here too.
```js
          if (locationTimeToLive) {
            const updatedAt = Date.now();
            const expiresAt = updatedAt + (locationTimeToLive * 60 * 1000); // convert to ms
            const lastLocation = { latitude, longitude, accuracy, updatedAt, expiresAt };
            Storage.setItem(Storage.LAST_LOCATION, JSON.stringify(lastLocation));
          }
```
Then when we read it out, we just need to check
```js
if (Date.now() < parseInt(lastLocation.expiresAt())) {
```